src/interfaces/EditInterface/EditInterface.css

```diff
@@ -1 +1,5 @@
 @import '../../common/styles/subHeadings';
+
+select[multiple] + span {
```

Comment:
seems, this should be copied to stripes

Reply:
let's wait feedback from POs if we are going to use Select like this)